docker-compose.override.yaml

```diff
@@ -0,0 +1,19 @@
+version: "3.7"
+
+services:
+  postgres:
```

Comment:
Is db volume preserved via some docker-compose magic?

Are there any mechanisms available to bootstrap db to some state?

Reply:
The DB volume sticks around until you do a `docker-compose down` or remove the container somehow.

If we wanted to mount a volume and mount it so that it's more permanent, we can, but that's a different use case, really.